packages/front-end/components/Experiment/DateResults.tsx

```diff
@@ -76,6 +75,7 @@ const DateResults: FC<{
       Array.from(new Set(metrics.concat(guardrails || [])))
         .map((metricId) => {
           const metric = getMetricById(metricId);
+          if (!metric) return undefined;
```

Comment:
style: `return` is equivalent to `return undefined` so an explicit undefined is probably not necessary.